src/components/Book.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import '../Styling/Book.css';
+
+const Book = (props) => {
+  /* eslint-disable react/prop-types */
```

Comment:
- [ ] Kindly don't disable the linter, if it is giving some error please fix it.